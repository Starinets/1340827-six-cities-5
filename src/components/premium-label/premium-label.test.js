import React from 'react';
import renderer from 'react-test-renderer';
import PremiumLabel from './premium-label';

describe(`Component rendered correctly -> PremiumLabel`, () => {
  it(`is true`, () => {
    const tree = renderer.create(
        <PremiumLabel
          isPremium={ true }
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`is false`, () => {
    const tree = renderer.create(
        <PremiumLabel
          isPremium={ false }
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
