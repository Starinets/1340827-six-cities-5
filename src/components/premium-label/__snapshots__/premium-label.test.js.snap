// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component rendered correctly -> PremiumLabel is false 1`] = `null`;

exports[`Component rendered correctly -> PremiumLabel is true 1`] = `
<div
  className="property__mark"
>
  <span>
    Premium
  </span>
</div>
`;
